src/cljs/zetawar/subs.cljs

```diff
@@ -456,8 +455,9 @@
 
 (deftrack compatible-armor-types-for-repair? [conn targeted-q targeted-r]
   (when-let [[selected-q selected-r] @(selected-hex conn)]
-    (game/compatible-armor-types-for-repair? @conn @(game conn) @(unit-at conn selected-q selected-r)
-                                                                @(unit-at conn targeted-q targeted-r))))
+    (game/compatible-armor-types-for-repair? @conn @(game conn)
```

Comment:
Hmm... maybe. I'm considering just merging it as is, doing a bit of cleanup myself, and then getting you to review that. The current state is pretty good and all the back and forth in comments can get a bit tedious. Master only gets deployed to dev.zetawar.com, so if something is still a little off it's not going to mess up anyone's games.